### DISCUSSION TITLE
Add SMTP support for Museum

### DISCUSSION THREAD

**Discussion by cadenza-k** [ente-io/ente#1137] [Enhancements]
Hi!

I saw there was a support for Zoho/transmail email campaigns for sending OTT and various notifications.

Sadly there is no standard SMTP support.

Is it possible to implement it alongside Zoho?

Thank you.

**Comment by mnvr** [comment]
We're adding support for more providers. In particular, we're considering [List Monk](https://listmonk.app). I can't say yet what the final form will look like, but if possible we'd try to make it such that it uses standard SMTP (it'll also make it easier for us also).

I can't say yet if that's the way it'll be for sure though. Sometimes some required integration features are not available over standard SMTP (but as I said, we'll try that approach first).

**Comment by mnvr** [comment]
@Eclipse2212 This has now been added. You can configure SMTP by providing the following configuration options in your configuration yaml.

```yaml
smtp:
    host:
    port:
    username:
    password:
```

The older method via Zoho is also supported, though we might deprecate that at some point in favor of the standard SMTP approach.

**Comment by roughnecks** [comment]
Am I correct in saying that emails received by a self-hosted server have this from address? `ente <verification@ente.io>`

**Reply by vishnukvmd** [reply]
Yes
